sidebars.js

```diff
@@ -427,4 +427,32 @@ module.exports = {
       ],
     },
   ],
+  tctlReferenceMain: [
+    {
+      type: "category",
+      label: "tctl commands",
+      collapsible: false,
```

Comment:
And here, by making it non-collapsible we get a nice static list -

Reply:
Though, on second thought - we might want to let this level be collapsible ... experimentation needed :)

Reply:
Oh, I like this very much! I left the top level as non-collapsible and made the command level collapsible but collapsed by default. The result makes it impossible to get lost in the tctl reference.

(Wait. Did I just hear someone say, "Hold my beer"? Naw. Must be imagining things.)